portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,131 @@
+@import url('https://fonts.googleapis.com/css2?family=Noto+Sans&display=swap');
+
+:root {
+    --g-red:  #ea4335;
+    --g-blue: #4285f4;
+    --g-green:  #34a853;
+    --g-yellow:  #fbbc05;
+}
+
+a {
+    text-decoration: none;
+    color: #4791ff;
+}
+
+hr {
+    margin-bottom: 3em;
+}
+
+body {
+    margin: 0;
+}
+
+h1 {
+    font-size: 3.5em;
+    margin-top: 0%;
+    margin-bottom: 2%;
+}
+
+#intro {
+    font-size: 2.2em;
+    margin-bottom: 5%;
+}
+
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  width: 850px;
+  margin-top: 5%;
 }
 
-#greeting-container {
+#fact-container {
   margin-top: 20px;
 }
+
+#pfp {
+    border-radius: 2%;
+    border: 2px solid black;
+}
+
+.my-button {
+    background-color: var(--g-green);
+    border-radius: 12px;
+    border: none;
+    color: white;
+    padding: 15px 32px;
+    text-align: center;
+    text-decoration: none;
+    cursor: pointer;
+    font-size: 20px;
+    font-weight: 600px;
+}
+
+.container {
+    font-family: 'Noto Sans', sans-serif;
+}
+
+.nav-wrapper {
+    display: flex;
+    justify-content: space-between;
+    padding: 1% 4%;
+    background-color: #2d2d2d;
+}
+
+.nav-left {
```

Comment:
Consider grouping together related selectors (all nav-* toghether and all .g-* toghether). Making assumptions about file structure makes it easier to skip/focus and not miss things when reading.